__tests__/api.test.js

```diff
@@ -0,0 +1,24 @@
+const request = require("supertest");
+const seedData = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection.js");
+const app = require("../api/server");
+
+beforeEach(async () => {
+  await seed(seedData);
+});
+
+afterAll(async () => {
+  await db.end();
+});
+
+describe("/api/topics", () => {
+  it("GET: 200 sends all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toEqual(seedData.topicData);
```

Comment:
Interesting test! This formula works well for the next ticket too, so it's interesting to see you apply it to this ticket. However, be aware that this will not work for other endpoints, and you need to check that the response has the correct key-value pairs that you expect :)